internal/injector/testdata/injector/wrap-errors/config.yml

```
@@ -0,0 +1,189 @@
+# Example configuration demonstrating the desired 'wrap-errors' functionality.
+# We expect returned errors are wrapped with dd-trace-go/instrumentation/errortrace
+# "errors" should also include popular, third-party error libraries
+
+%YAML 1.1
+---
+aspects:
+  - id: "wrap-standard-errors"
+    join-point:
+      function-body:
+        function:
+          - result-implements: "error"
+    advice:
+      - prepend-statements:
+          imports:
+            errortrace: github.com/DataDog/dd-trace-go/v2/instrumentation/errortrace
+            fmt: fmt
+          template: |-
+            {{- $ret := .Function.ResultThatImplements "error" -}}
```

```suggestion
            {{- $ret := .Function.FinalResultImplements "error" -}}
```

This could better for starters

---

Maybe `.Function.LastResultThatImplements`? 👀